lesson3/ToDoList2/app/src/main/java/erica/todolist2/MainActivity.java

```diff
@@ -55,14 +56,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
+    /**
+     * method replaces the current fragment with the fragment that is given as an input
+     */
     public void replaceFragment(Fragment fragment) {
-        /**method replaces the current fragment with the fragment
-            that is given as an input */
-        FragmentManager manager;                                                                    //initializes manager as FragmentManager
-        FragmentTransaction transaction;                                                            //initializes transaction as FragmentTransaction
-
-        manager = getSupportFragmentManager();
-        transaction = manager.beginTransaction();
+        FragmentManager manager = getSupportFragmentManager();
+        FragmentTransaction transaction = manager.beginTransaction();
```

Comment:
Instead of declaring the variable manager and transaction, and then setting them equal to something in another line, do it in one line!  No need to use extra lines.  More lines makes your code less readable!